src/shogun/lib/auto_initialiser.h

```diff
@@ -124,14 +124,14 @@ namespace shogun
 								++idx;
 							}
 						}
-						width = GaussianKernel::to_log_width(linalg::median(result));
+						width = std::sqrt(linalg::median(result) / 2.0);
```

Comment:
@karlnapf I think this is now the correct computaion? I think before it wasn't quite right, it should've been `GaussianKernel::to_log_width(std::sqrt(linalg::median(result) / 2.0)))`?